LogMonitor/src/LogMonitor/LogFileMonitor.cpp

```diff
@@ -1696,12 +1698,26 @@ void LogFileMonitor::WriteToConsole( _In_ std::wstring Message, _In_ std::wstrin
         if (msg.size() > 0) {
             // escape backslashes in FileName
             auto fmtFileName = Utility::ReplaceAll(FileName, L"\\", L"\\\\");
-            // sanitize msg
-            Utility::SanitizeJson(msg);
-            auto log = Utility::FormatString(logFmt, msg.c_str(), fmtFileName.c_str());
-            logWriter.WriteConsoleLog(log);
-        }
+            std::wstring formattedlog, logFmt;
+            if (Utility::CompareWStrings(m_logFormat, L"JSON"))
+            {
+                logFmt = L"{\"Source\": \"File\",\"LogEntry\": {\"Logline\": \"%s\",\"FileName\": \"%s\"},\"SchemaVersion\":\"1.0.0\"}";
+                // sanitize message
+                Utility::SanitizeJson(msg);
+
+                formattedlog = Utility::FormatString(
+                    logFmt.c_str(),
+                    msg.c_str(),
+                    fmtFileName.c_str());
+            } else {
+                logFmt = L"<Source>File</Source><LogEntry><Logline>%s</Logline><FileName>%s</FileName></LogEntry>";

```

Comment:
schema: same here, `<LogEntry>` to go to the beginning, as the root node.

Reply:
Addressed